api/src/pcapi/core/bookings/api.py

```diff
@@ -460,15 +462,29 @@ def cancel_booking_by_offerer(booking: Booking) -> None:
 
 
 def cancel_bookings_from_stock_by_offerer(stock: offers_models.Stock) -> list[Booking]:
-    cancelled_bookings = _cancel_bookings_from_stock(stock, BookingCancellationReasons.OFFERER)
+    cancelled_bookings = _cancel_bookings_from_stock(
+        stock, BookingCancellationReasons.OFFERER, typing.cast(bool, stock.offer.isEvent)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Pas besoin de cast ici
On a déjà un booléen